### DISCUSSION TITLE
Getting error when transfer data from client to miner

### DISCUSSION THREAD

**Discussion by arijitAD** [filecoin-project/lotus#3426] [Ideas] [answered]
I have implemented [go-fil-markets](https://github.com/filecoin-project/go-fil-markets). 
Implementation: https://github.com/ChainSafe/go-fil-markets-interface
I am able to start a deal but the client is not able to transfer data to the miner.
Getting the following error.
```
2020-08-28T16:51:03.310+0530	INFO	graphsync responsemanager/responsemanager.go:445	response failed: %!w(*errors.errorString=&{request not valid})
```
This the happening because the request is not validated. `result.IsValidated` is false in the below line.

https://github.com/ipfs/go-graphsync/blob/bd2d62f57b0f1590dd657ff8b1f3e08848f3f8a4/responsemanager/queryexecutor.go#L129

Any pointers how I may be able to fix the issue?

**Comment by jennijuju** [comment]
Doesnt seems like a lotus issue, but is something interesting for a discussion .

**Comment by SgtCoin** [comment]
@arijitAD Are you still having issues with this? If so, I would like to help, otherwise we can archive this discussion for future reference.

**Comment by arijitAD** [comment, marked as answer]
This issue is resolved. You can close it.
